src/Hyperwallet/Util/ApiClient.php

```diff
@@ -143,10 +144,14 @@ public function doGet($partialUrl, array $uriParams, array $query) {
     private function doRequest($method, $url, array $urlParams, array $options) {
         try {
             $uri = new UriTemplate();
+            if (!isset($options['headers'])) {
+                $options[] = array('headers' => array());
+            }
+            if (!isset($options['headers']['Accept'])) {
+                $options['headers'][] = array('Accept' => 'application/json');
+            }
+            $options['headers']['Accept'] = 'application/json';
             if ($this->isEncrypted) {
-                if (!isset($options['headers'])) {
-                    $options[] = array('headers' => array());
-                }
                 if (!isset($options['headers']['Content-Type'])) {
                     $options['headers'][] = array('Content-Type' => 'application/jose+json');
                 }
```

Comment:
in this `if-clause` if `isEncrypted = true` would it be better to force header to be the ff:
`$options['headers']['Accept'] = 'application/jose+json';`
`$options['headers']['Content-Type'] = 'application/jose+json';`

meaning we don't need line 155 to check if is set or not we just directly override to use encrypted content and we will expect encrypted response
that is our sdk needs only to Accept and Content-Type be 'application/jose+json' in this case

Does the information above make sense? Moreover server side support this scheme so we are good
btw what is the difference between 
`$options['headers'][] = array('${mime-type'} => '${type}');` and ` $options['headers']['${mime-type}'] = '${type}'` ?
for me the latter makes more sense?

Reply:
Fixed